src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -154,7 +156,7 @@ function StripePaymentForm({
               // SDN failure: redirect to Ecommerce SDN error page.
               setLocation(`${getConfig().ECOMMERCE_BASE_URL}/payment/sdn/failure/`);
             } else if (errorData && errorData.sku_error) {
-              skuErrorDispatcher();
+              basketChangedErrorDispatcher();
```

Comment:
The variables were changed from "sku error" related to "basket changed", are you planning to keep the `sku_error` variable in ecommerce the same?

Reply:
I am planning to leave sku_error as is in ecommerce, as a sku_error is just one type of error that indicates the situation where a basket has changed